src/applications/claims-status/components/ClaimCard/ClaimCardLink.jsx

```diff
@@ -16,7 +16,7 @@ export default function ClaimCardLink({
       onClick={onClick}
     >
       {text}
-      <i aria-hidden="true" />
+      <va-icon icon="chevron_right" size={3} aria-hidden="true" />
```

Comment:
I guess my only question so far is regarding the size of these icons. Based on the screenshots, it looks like they were all slightly smaller before?

Reply:
Size 3 is the closest that I could get to the original size (where its a little bigger then the text), size 2 makes it smaller then the text and I didnt think we would want that.

Reply:
Ok, that makes sense